### DISCUSSION TITLE
Adding a /page to create a direct link to a new page.

### DISCUSSION THREAD

**Discussion by MathieuSibret-UCLL** [outline/outline#6135] [Ideas]
It would be really nice to have a '/page' command for seamless creation of new pages. 

When the '/page' command is executed, it could trigger a prompt on a new document creation page where users can conveniently fill in the details for the new sub-page. Upon creating the sub-page, the title of the new page would automatically appear on the original page where the '/page' command was initiated. Clicking on this title would then redirect users to the corresponding sub-page. 

While a similar functionality is achievable by clicking and dragging a page from the sidebar to the document, refining it with a dedicated '/page' command could significantly enhance the user experience. 

As an alternative, achieving a similar result is possible with the '/link' command. Users can input the desired title and select 'create new doc.' Although this option exists and is useful, integrating the '/page' command could accelerate the process with fewer user interactions, contributing to an enhanced overall experience.

In addition, it would be beneficial to enhance the visual presentation of links and pages. Instead of a simple underline, consider incorporating a subtle contour or a small icon preceding the link. This visual distinction could provide a more polished and intuitive look, making it easier for users to differentiate between regular text and links, especially when they represent documents or redirect to different pages.

**Comment by SaadBazaz** [comment]
The current link leaves much to desire. It can have the icon with it, and a bit of a button-like view instead of just an underlined text.

**Reply by ArnaudMiguet** [reply]
A Button like view would be great (embeddable into tables too). You could then bind some webhooks to thoses buttons and get some automation going :)
